starlite_jwt/jwt_auth.py

```diff
@@ -165,6 +177,17 @@ def login(
         return Response(
             content=response_body,
             headers={self.auth_header: encoded_token},
+            cookies=[
```

Comment:
Given my suggestion to use a subclass - I would suggest to also subclass the Middleware. If not, it should check the type of config it has here.